installation/licensing/license-errors-warnings.md

```diff
@@ -28,14 +28,16 @@ When using Telerik UI for .NET MAUI in a project with an expired or missing lice
 
 | Error or Condition                              | Message Code | Solution                                                              |
 |-------------------------------------------------|--------------|-----------------------------------------------------------------------|
-| `No license key is detected` | TKL002 | [Set up a license key]({%slug set-up-your-license%}) to activate the UI controls and remove the error message. |
-| `Invalid license key` | TKL003 | [Download a new license key]({%slug set-up-your-license%}#downloading-the-license-key) and use it to activate the UI controls and remove the error message. |
-| `Your subscription license has expired.` | TKL103; TKL104 | Renew your subscription and [download a new license key]({%slug set-up-your-license%}#downloading-the-license-key). |
-| `Your perpetual license is invalid.` | TKL102 | You are using a product version released outside the validity period of your perpetual license. To remove the error message, do either of the following: |
+| `No Telerik and Kendo UI License file found` | TKL002 | [Set up a license key]({%slug set-up-your-license%}) to activate the UI controls and remove the error message. |
```

Comment:
Are these three consecutive errors missing a period when shown to the user, as well?